massa-node/base_config/config.toml

```diff
@@ -52,7 +52,7 @@
     # whether to enable mTLS
     enable_mtls = false
     # bind for the Massa gRPC API
-    bind = "0.0.0.0:33037"
+    bind = "127.0.0.1:33037"
```

Comment:
We got a hell of issue because of this. @Leo-Besancon  @modship do you remember why exactly ?

Reply:
This will cause the API do not be seen the people outside of your local network. We shouldn't to that.

Reply:
i think 0.0.0.0 accept traffic on all interface instead of 127.0.0.1

Reply:
Yes, the private API binds to localhost (127.0.0.1 or [::1] in IPv6), but public APIs should bind to 0.0.0.0 or [::] in IPv6.

Reply:
Security wise I think it's better for now to only listen on localhost and let the user open it if they want to.

Reply:
> Security wise I think it's better for now to only listen on localhost and let the user open it if they want to.
both listening on localhost

Reply:
> > Security wise I think it's better for now to only listen on localhost and let the user open it if they want to.
> 
> both listening on localhost

Not sure to understand this? :-/